packages/expo-ui/components/Progress/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { requireNativeView } from 'expo';
+import { StyleProp, ViewStyle } from 'react-native';
+
+export type ProgressElementColors = {
+  /**
+   * Track color.
+   *
+   * @platform android
+   */
+  trackColor?: string;
+};
+
+export type ProgressProps = {
+  /**
+   * Custom styles for the progress component.
+   */
+  style?: StyleProp<ViewStyle>;
+  /**
+   * The current progress value of the slider. This is a number between 0 and 1.
+   */
+  progress?: number | null;
+  /**
+   * Progress color.
+   */
+  color?: string;
```

Comment:
We can use `ColorValue` from `react-native` to support `PlatformColor` (also applies to `trackColor`)